sale_cancel_reason/security/ir.model.access.csv

```diff
@@ -0,0 +1,3 @@
+id,name,model_id:id,group_id:id,perm_read,perm_write,perm_create,perm_unlink
+access_sale_order_cancel_reason_user,access_sale_order_cancel_reason user,model_sale_order_cancel_reason,sales_team.group_sale_salesman,1,0,0,0
+access_sale_order_cancel_reason_manager,access_sale_order_cancel_reason manager,model_sale_order_cancel_reason,sales_team.group_sale_manager,1,1,1,1
```

Comment:
missing security for wizard i.e. sale.order.cancel
In v14, we need to manage access rights for TransientModels too